README.md

```diff
@@ -53,54 +53,45 @@ We relabled the WIDER Face train set using [RetinaFace](https://github.com/deepi
 
 ## Training
 ```Shell
-cd $TRAIN_ROOT/tasks/task1/
-python train.py
+python train.py -c ./config/yufacedet.yaml -t demo 
 ```
 
 ## Detection
 ```Shell
-cd $TRAIN_ROOT/tasks/task1/
-python detect.py -m weights/yunet_final.pth --image_file=filename.jpg
+python detect.py -c ./config/yufacedet.yaml -m weights/yunet_final.pth --target filename.jpg 
 ```
 
 ## Evaluation on WIDER Face
 1. Build NMS module.
     ```shell
-    cd $TRAIN_ROOT/src/widerface_eval
+    cd tools/widerface_eval
     python setup.py build_ext --inplace
```

Comment:
cython-based widerface evaluation has been replaced with a numpy implemetation. See here: https://github.com/ShiqiYu/libfacedetection.train/blob/master/src/eval.py. Please keep this implementation instead of using the old cython-based one.